### PR TITLE
Port KiCad v6 installers for eSim

### PR DESCRIPTION
**Pull request by PSR0001**
### Purpose
eSim installer support for KiCad V6.0.11.

Related PR: https://github.com/FOSSEE/eSim/pull/255

### CODE REVIEW

File: Ubuntu/install-eSim.sh [hunk 1]
```diff
@@ -183,45 +183,49 @@ function installDependency
...
 
     #Copy fp-lib-table for switching modes
-    cp -r library/supportFiles/fp-lib-table ~/.config/kicad/
-    cp -r library/supportFiles/fp-lib-table-online ~/.config/kicad/
+    cp -r library/supportFiles/fp-lib-table ~/.config/kicad/6.0
+    cp -r library/supportFiles/fp-lib-table-online ~/.config/kicad/6.0
     echo "fp-lib-table copied in the directory"
 
+    # copy sym-lib-table for eSim-custom symbols 
+    cp -r library/supportFiles/sym-lib-table ~/.config/kicad/6.0
```
[the remark this answers is not in the document]
Comment (PSR0001) [reply Ubuntu/install-eSim.sh:202]: Sir, `sym-lib-table` location is updated. see #255

File: Windows/esim-setup-script.nsi [hunk 2]
```diff
@@ -257,22 +257,23 @@ Section -InstallKiCad
     ;CopyFiles "$INSTDIR\eSim\library\kicadLibrary\library\*" "$INSTDIR\KiCad\share\kicad\library\"
 
     ;Copy KiCad library made for eSim
-    CopyFiles "$INSTDIR\eSim\library\kicadLibrary\kicad_eSim-Library\*" "$INSTDIR\KiCad\share\kicad\library\"
+    CopyFiles "$INSTDIR\eSim\library\kicadLibrary\kicad_eSim-Library\*" "$INSTDIR\KiCad\share\kicad\symbols\"
     
-    CopyFiles "$INSTDIR\eSim\library\kicadLibrary\modules\*" "$INSTDIR\KiCad\share\kicad\modules\"
-
-    CopyFiles "$INSTDIR\eSim\library\kicadLibrary\template\*" "$INSTDIR\KiCad\share\kicad\template\"
+    ;No need to copy
```
[the remark this answers is not in the document]
Comment (PSR0001) [reply Windows/esim-setup-script.nsi:262]: In Windows, [kicad-6.0.11-i686.exe](https://github.com/FOSSEE/KiCad-eSim) already comes with the `fp-lib-table`. For the Windows installer, it is no longer needed to copy `fp-lib-table`.


### Related PR: #255 https://github.com/FOSSEE/KiCad-eSim/pull/11

[the remark this answers is not in the document]
Comment (rahulp13) [reply Windows/esim-setup-script.nsi:262]: @PSR0001 Add the comments/notes to the README file under the Windows directory.

[the remark this answers is not in the document]
Comment (PSR0001) [reply Windows/esim-setup-script.nsi:262]: @rahulp13 Readme Updated.

**Review by rahulp13 — APPROVED** [review]

**Review by Eyantra698Sumanto — APPROVED** [review]

**Pull request merged**